report/src/system_evaluation/results.tex

```diff
@@ -32,28 +32,40 @@ \section{System evaluation}\label{sec:experiment_results}
+    \end{tabular}}
+    \caption{Evaluation results from Lenovo TabM10 FHD Plus showing the number of correct classifications out of a possible ten for each $k$-value}
+    \label{lst:resultsTablet}
+\end{table}
 
-To create an environment that more accurately represents a meeting room in use, several devices were used for testing. 
-The method for collecting results from these tests mirrored previous ones, by making $10$ classifications on both a Samsung Galaxy A53, a Samsung Galaxy S20 phone, and a Lenovo TabM10 FHD Plus tablet. 
-The classification results were gathered in the locations shown on Figure \ref{fig:experiment_room}
-During the experiment, k-values of $2$, $3$, and $4$ were used.  
-The results of the experiment can be seen in Tables \ref{lst:resultsPhone}, \ref{lst:resultsPhoneS20}, and  \ref{lst:resultsTablet}.
-Table \ref{lst:resultsPhone_precision} summarizes the experiment's results, indicating the percentage of true positive classifications. 
+On Table \ref{lst:resultsPhone_precision} see that a lower $k$-value results in a larger uncertainty in the classification.
+However, it is worth noticing that for measuring points \textit{B} and \textit{J}  (see Figure \ref{fig:experiment_setup}) a large uncertainty in classification is observed.
+We suspect that this is a consequence of the glass pane next to the door having a much lower absorbation of radio frequencies than the door and the wall next to it. 
+This low absorbation results in the two data points next to the pane having similar distances to both map values labelled as inside and some as outside, therefore giving uncertain classifications.
+
+Similarly, for all devices, when performing the classification in locations H, I, J, K, and L shows, the results are vastly different from the other locations.
+These locations are all outside of the room. 
+We suspect that this can be due to two things.
+First and foremost, it might be due to the created map being unreliable. 
+When creating the map, the wall separating the room and the common area might cause large fluctuations in the strength of the received signal. 
+Therefore, classification using the map becomes unreliable.
+We tried to accomodate this by enforcing all map entries to have a standard deviation of 6.5 \todo{units and 6.5}. 
+Enforcing a lower standard deviation of the map might alleviate these issues.
+However, this will increase the time it takes to create the map, as more samples will be required if the signal fluctuates.
+Furthermore, detecting these signal outliers might be difficult if the signal often fluctuates by high degrees.      
+
+Secondly, by having placed beacons only inside of the meeting room, and by having more classification data representing locations inside the meeting room, the classification results might be skewed towards \textit{inside} classifications. 
```

Comment:
Shouldn't this be closer to `first`? Otherwise rephrase. They're too far apart.
```suggestion
Secondly, by having placed beacons only inside of the meeting room, and by having more classification data representing locations inside the meeting room, the classification results might be biased towards \textit{inside} classifications. 
```

Reply:
```suggestion
Furthermore, by having placed beacons only inside of the meeting room, and by having more classification data representing locations inside the meeting room, the classification results might be biased towards \textit{inside} classifications. 
```

Reply:
Readers would probably wonder where the second thing went when we say "two things" followed by "first" and no "second".